lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -22,10 +22,20 @@ def setUp(self):
         CertificateGenerationConfiguration.objects.create(enabled=True)
 
     def test_cert_generation_enabled_for_self_paced(self):
-        """ Verify the signal enable the self-generated certificates by default for
+        """ Verify the signal enable the self-generated certificates for
         self-paced courses.
         """
         self.assertFalse(certs_api.cert_generation_enabled(self.course.id))
 
         _listen_for_course_publish('store', self.course.id)
         self.assertTrue(certs_api.cert_generation_enabled(self.course.id))
+
+    def test_cert_generation_disabled_for_instructor_paced(self):
+        """ Verify the signal disable the self-generated certificates for
```

Comment:
See http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings

Like this:
```
def foo(a, b):
    """
    Computes the foo of a and b.
    """
```